### PR TITLE
increases dual wielded energy weapon spread

### PR DESCRIPTION
**Pull request by ghost**
Current dual wielded disablers/lasers have very little spread, and 1-2 shots shoot straight enough to hit the tile you are clicking on at range 7 a majority of the time, which is pretty high.

With the new spread their effective range in now smaller, around 3-4 tiles.

Spread unchanged for dual wielding on the other acquirable guns.

:cl:
balance: Increased spread on dual wielded energy weapons.
/:cl:

### CODE REVIEW

**Comment by Shadowflame909** [comment]
people say nerf

but this might change the game of all those pro-dodgers with higher ping

**Comment by Tlaltecuhtli** [comment]
it is already shit, you dual wield within 2-3 tiles to get people, maybe lower the 40 shot mag disablers get so spraying and praying wont be a valid strategy

**Comment by zxaber** [comment]
The effort here is to nerf dual-weilded disablers without nerfing the single-wielded disabler .

**Comment by 81Denton** [comment]
Needs a changelog

File: code/modules/projectiles/gun.dm [hunk 1]
```diff
@@ -227,7 +228,7 @@
 			if(G == src || G.weapon_weight >= WEAPON_MEDIUM)
 				continue
 			else if(G.can_trigger_gun(user))
-				bonus_spread += 24 * G.weapon_weight
+				bonus_spread += dual_wield_spread
```
Comment (Fikou) [inline code/modules/projectiles/gun.dm]: why isnt this dual_wield_spread * G.weapon_weight

Comment (py01) [reply code/modules/projectiles/gun.dm]: the only values of weapon weight are 1, 2, and 3. Only weapons with a weapon weight of 1 can dual wield, so G.weapon_weight is only ever equal to 1. So I removed it to improve code clarity

**Pull request merged**